src/integrationTest/java/uk/gov/hmcts/reform/civil/controllers/DashboardBaseIntegrationTest.java

```diff
@@ -29,7 +29,7 @@ Client feignClient() {
         }
     }
 
-    protected static CallbackParams callbackParams(CaseData caseData) {
+    public static CallbackParams callbackParams(CaseData caseData) {
```

Comment:
why public, this class is being extended, so sub class can directly access with protected